docs/developer/0_prerequisites.md

```diff
@@ -0,0 +1,31 @@
+# 💼 Prerequisites
+
+Here's everything you'll need to develop on ReVanced Manager
+
+## 🤝 Recommended environment
+
+- Any environment that is capable of running the latest Android Studio, latest Android SDK, and JDK 17 with at least 4 GB of memory for coding
+- Any devices with preferably the latest Android version or at least higher than the [`minSdk`](/app/build.gradle.kts) for testing
+
+### Additional Requirements
+
+- A device with root capabilities using the latest version of [KernelSU](https://github.com/tiann/KernelSU) or [Magisk](https://github.com/topjohnwu/Magisk)
```

Comment:
This is not required unless you are working on root features.

Reply:
This could be reworded into optional requirements signalling that this requirement does not have to be met.